src/llaminator.ts

```diff
@@ -35,15 +36,27 @@ interface LlaminatorElements {
   select: LlamaSelectFab;
 }
 
+interface Layout {
```

Comment:
This should be in its own file, so that `LlamaVerticalScrollLayout` can implement it as an interface.

Reply:
For more context - typing in TypeScript can be a bit implicit, e.g. the following is allowed:

```typescript
type Foo = { a: number };
type Bar = { a: number, b: number };

function output(data: Foo) {}

const a: Foo = { a: 16 };
const b: Bar = { a: 12, b: 24 };

output(a);  // OK
output(b);  // also OK - because the conditions of the "Foo" type are met
```

Because of this, without `LlamaVerticalScrollLayout` implementing the interface, we'd get an error in this file on line 71 if the expected interface ever changes, whereas the `LlamaVerticalScrollLayout` file will pretend to be completely OK. That's super confusing.

Reply:
I guess I'm too used to the Go way